libs/traits/src/connectors.rs

```diff
@@ -21,6 +21,15 @@ pub trait Codec: Sized {
 	fn deserialize<I: Input>(input: &mut I) -> Result<Self, codec::Error>;
 }
 
+/// The trait required for forwarding messages.
+pub trait Router {
+	type Message;
+
+	type Sender;
+
+	fn forward(&self, sender: Self::Sender, message: Self::Message) -> DispatchResult;
```

Comment:
On the `Solidity` side we named this `send`, which I personally prefer over `forward`. Can we align here and name it `send` as well?